conf/mongodb/create_indexes.js

```diff
@@ -1,64 +1,64 @@
-db.products.createIndex({brands_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({categories_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({labels_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({packaging_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({origins_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({manufacturing_places_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({emb_codes_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({ingredients_tags:1,"last_modified_t": -1}, { background: true });
 db.products.createIndex({additives_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({vitamins_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({minerals_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({amino_acids_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({nucleotides_tags:1,"last_modified_t": -1}, { background: true });
 db.products.createIndex({allergens_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({traces_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({nova_groups_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({nutrition_grades_tags:1,"last_modified_t": -1}, { background: true });
-db.products.createIndex({languages_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({amino_acids_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({brands_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({categories_properties_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({categories_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({checkers_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({cities_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({code: 1}, { background: true });
+db.products.createIndex({codes_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({correctors_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({countries_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({countries_tags:1,"popularity_key": -1}, { background: true });
+db.products.createIndex({"created_t": -1}, { background: true });
+db.products.createIndex({creator:1,"last_modified_t": -1}, { background: true });
 db.products.createIndex({creator_tags:1,"last_modified_t": -1}, { background: true });
+db.products.createIndex({data_quality_bugs_tags:1,"last_modified_t": -1}, { background: true });
```

Comment:
quality tags indexes might be a good candidate for grouping, should we need to free indexes